src/loader/extension.h

```diff
@@ -0,0 +1,13 @@
+#ifndef TIMESCALEDB_EXTENSION_H
+#define TIMESCALEDB_EXTENSION_H
+#include <parser/analyze.h>
+
+#define EXTENSION_NAME "timescaledb"
+
+
+extern void extension_check(void);
```

Comment:
Not sure I like any extern functions in this shared library. What if other libraries have functions with same name? I see 2 options:
1) prefix these with `ts_loader` or something
2) make these static and move everything into init.c
I think I prefer 2 even though the code is a bit less organized.